generation/2d_autoencoderkl/2d_autoencoderkl_tutorial.ipynb

```diff
@@ -0,0 +1,782 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import time
</pre><p>Need to import tempfile here as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>